persistence/sql/migrations/sql/20220607000001000000_hydra_login_challenge.sqlite3.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "selfservice_login_flows" ADD COLUMN "hydra_login_challenge" char(36) NOT NULL DEFAULT "00000000-0000-0000-0000-000000000000";
+ALTER TABLE "selfservice_registration_flows" ADD COLUMN "hydra_login_challenge" char(36) NOT NULL DEFAULT "00000000-0000-0000-0000-000000000000";
```

Comment:
Wouldn't a `CHAR(36) NULL` not make more sense here?

Reply:
I used not null because it is one fewer special case to handle (zero UUID is not a valid value), but I changed it to nullable since you commented on it.